security/vulnerability-management.md

```diff
@@ -2,84 +2,91 @@ Vulnerability Management Policy
-investigation to verify that the vulnerability cannot be exploited from our code.
+When a vulnerability is introduced by third party libraries or applications and cannot be removed, the developer should 
+carry out an investigation to verify that the vulnerability cannot be exploited from our code.
 
-Any waiver must be documented and approved by relevant project managers. Exceptions should only be granted if a strong justification is
-provided.
+Any waiver must be documented and approved by relevant project managers. Exceptions should only be granted if a strong 
+justification is provided.
 
 In those cases, we might need to put a waiver in our code to remove that  flag
 
-- *1 year*: This is a false positive. However, there is a small risk that the use of this library changes, rendering it a real risk. An
-  annual review is advised to confirm we keep using the library in the same context.
-- *6 months*: The current state and use of the library are not vulnerable.  However, we need to verify that we keep using the best practices.
-- *3 months*: The vulnerability comes from a third party which has acknowledged it and verified that their code is not vulnerable. We might
-  need to wait for them to release a fix.
+- *1 year*: This is a false positive. However, there is a small risk that the use of this library changes, rendering it 
+  a real risk. An annual review is advised to confirm we keep using the library in the same context.
+- *6 months*: The current state and use of the library is not vulnerable.  However, we need to verify that we keep using
+  the best practices.
+- *3 months*: The vulnerability comes from a third party which has acknowledged it and verified that their code is not 
+  vulnerable. We might need to wait for them to release a fix.
 
 ## 10. Monitoring and Improvement
-- **Training and Awareness**: Provide ongoing training for developers and other relevant staff on secure coding practices and emerging
-  threats.
+- **Training and Awareness**: Provide ongoing training for developers and other relevant staff on secure coding 
+  practices and emerging threats.
 - **Policy Review**: This policy will be reviewed and updated annually or as needed to ensure its effectiveness.
-- **Iterative approach**: This policy will be defined as an iterative approach to tackle vulnerabilities. VisitScotland counts with
-  numerous repositories and many of them might contain vulnerabilities due to the lack of maintenance. This process will take place gradually
-  and trying to minimize the impact in the development cycle.
+- **Iterative approach**: This policy will be defined as an iterative approach to tackle vulnerabilities. VisitScotland 
+  has numerous repositories and many of them might contain vulnerabilities due to the lack of maintenance. This process 
+  will take place gradually and trying to minimise the impact in the development cycle.
```

Comment:
"trying" should be **try**   or    reword to **will take place gradually, trying to**